assets/node-exporter/daemonset.yaml

```diff
@@ -10,6 +10,10 @@ spec:
   selector:
     matchLabels:
       app.kubernetes.io/name: node-exporter
+  updateStrategy:
```

Comment:
@sdodson changing assets directly won't work. You'll have to update the underlying jsonnet as the library for the node_exporter daemonset is an open source upstream project.

Please contribute the change upstream in https://github.com/prometheus-operator/kube-prometheus/blob/master/jsonnet/kube-prometheus/node-exporter/node-exporter.libsonnet.

Please create an issue upstream in https://github.com/prometheus-operator/kube-prometheus and let's discuss with the community there. We can then, once merged into master, cherry-pick into releases and then vendor downstream.

Reply:
ok, will work on that over the coming days, thanks

Reply:
@sdodson let me know if you need help in the upstream contribution, and thank you! :tada:

Reply:
Opened https://github.com/prometheus-operator/kube-prometheus/pull/661